src/Mailgun/Resource/Api/SimpleResponse.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright (C) 2013-2016 Mailgun.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+namespace Mailgun\Resource\Api;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class SimpleResponse
+{
+    /**
+     * @var string
+     */
+    private $message;
+
+    /**
+     * Only set when API rate limit is hit and a rate limit response is returned.
+     *
+     * @var int
+     */
+    private $retrySeconds = null;
+
+    /**
+     * Only set on calls such as DELETE /v3/domains/.../credentials/<user>.
```

Comment:
In the case of `DELETE /domains/<domain>/credentials/<login>`, $spec is the login name/mailbox name of the credential pair that was deleted.